### PR TITLE
@accounts/error : new AccountsError package with message formatting

### PR DESCRIPTION
**Pull request by Aetherall**
This is a first implementation of the AccountsError package.

````ts
new AccountsError('PasswordService','authenticate','no user match tokens')
// message => '[ Accounts - PasswordService ] authenticate : no user match tokens'

new AccountsError('no user match tokens')
// message => 'no user match tokens'
````

As the error code and login info is not used yet, I removed the code concerning them so we can implement it accordingly with the current state

### CODE REVIEW

File: packages/error/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    },
+    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx)$",
+    "moduleFileExtensions": [
+      "ts",
+      "js"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/js-accounts/accounts/tree/master/packages/server"
```
Comment (davidyaha) [inline packages/error/package.json]: https://github.com/js-accounts/accounts/tree/master/packages/error

Comment (Aetherall) [reply packages/error/package.json]: my bad, will be fixed in the next minutes

**Comment by pradel** [comment]
@Aetherall is there a way to get 100% test coverage for the error package in oder to not loose `1.78%` of code coverage for the global repository?

**Comment by Aetherall** [comment]
Actually the coverage missing comes from

````ts
if (typeof Error.captureStackTrace === 'function') {
  Error.captureStackTrace(this, this.constructor);
} else { 
  this.stack = (new Error(message)).stack; 
}
````

It would be a string comparison of something that is not predictable
Maybe there is an other way to do it I am not aware of ?

**Comment by Aetherall** [comment]
Can I merge This ?

**Comment by pradel** [comment]
You will need to prettier before I think and let's wait for @davidyaha review

**Comment by davidyaha** [comment]
just figure out the tests but that is good to go IMO

**Pull request merged**